packages/suite/src/actions/wallet/fiatRatesActions.ts

```diff
@@ -306,6 +306,26 @@ export const updateTxsRates = (account: Account, txs: AccountTransaction[]) => a
     }
 };
 
+const updateMissingTxRates = (symbol?: Network['symbol']) => (
+    dispatch: Dispatch,
+    getState: GetState,
+) => {
+    const { accounts, transactions } = getState().wallet;
+    accounts.forEach(account => {
+        if (symbol && account.symbol !== symbol) {
+            return;
+        }
```

Comment:
`symbol` is not required by the function logic, that's why it is optional.

if you won't pass the parameter all you will trigger the update logic for all accounts, if you pass it than it will run only for accounts belonging under the `symbol` network